pylibczi/CziScene.py

```diff
@@ -312,7 +319,7 @@ def read_scene_image(self):
             print('Loading czi image for scene %d' % (load_scene+1,)); t = time.time()
```

Comment:
I know you didn't change this but if we could not use prints and instead use logging that would be cool.